core/packages/test/scripts/set-env.sh

```diff
@@ -37,42 +37,9 @@ address_for()
     jq -r ".contracts.${1}.address" "$output_dir/contracts.json"
 }
 
-kill_trap() {
-    trap - SIGTERM
-    pkill -P $$
-}
-
 kill_all() {
-    kill_trap
-    kill_chains
-    kill_relayer
-}
-
-kill_chains() {
-    echo "Killing chains"
-    kill_polkadot
-    kill_ethereum
-}
-
-kill_ethereum() {
-    pkill -9 -f lodestar
-    pkill -9 geth
-}
-
-kill_polkadot() {
-    pkill -9 polkadot
-    pkill -9 snowbridge-test-node
-    pkill -9 snowbridge
-    pkill -9 -f polkadot-launch
-    pkill -9 zombienet
-}
-
-kill_relayer() {
-    echo "Killing relayer"
-    kill_trap
-    sleep 3
-    pkill -9 snowbridge-relay
-    sleep 1
```

Comment:
IMHO starting and killing chains separately could be helpful in dev stage, 
e.g: when we only change some code/logic in parachain, then no need to restart ethereum and deploy contracts again

Reply:
Sure, can revert my change.